css/media.css

```diff
@@ -0,0 +1,388 @@
+  }
+
+  .link-none {
+    display: none;
+  }
+
+  .welcome-page__block {
+    width: 50%;
+    margin: 0;
+  }
+}
+
+@media (max-width: 768px) {
+  .header {
+    box-sizing: border-box;
+    padding: 24px 33px 0 15px;
+  }
+
+  .header__menu {
+    display: none;
+  }
+
+  .link-none {
+    display: none;
+  }
+
+  .link__img {
+    width: 35.23px;
+    height: 22.02px;
+    margin-left: auto;
+    margin-right: 0;
+  }
+
+  .car__title {
+    margin-top: 80px;
+  }
+
+  .car__subtitle {
+    position: relative;
+    bottom: 10px;
+    padding-left: 16px;
+  }
+
+  .car {
+    background-size: 109%;
+    background-position: 15% 50%;
+  }
+
+  .car__tsl {
+    width: 91%;
+    position: relative;
+    left: 10px;
+    top: 65px;
+  }
+
+  .welcome-page {
+    margin-top: 104px;
+    display: block;
+  }
+
+  .welcome-page__block{
+    width: auto;
+  }
+
+  .welcome-page__1 {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .welcome-page__button-image {
+    width: 30px;
+    height: 30px;
+  }
+
+  .welcome-page__section {
+    margin-left: 0;
+    margin-right: 68px;
+  }
+
+  .welcome-page__speed-sec {
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  .welcome-page__driver-max {
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  .welcome-page__speed-car {
+    width: 120%;
+    padding: 18px 0 0 10px;
+  }
+
+  .discrip-mob {
+    margin-top: 2px;
+    margin-left: 38px;
+  }
+
+  .discrip-sek {
+    padding-left: 5px;
+    margin-bottom: 10px;
+    position: relative;
+    bottom: 8px;
+  }
+
+  .welcome-page__drive_img {
+    margin-right: 14px;
+    margin-top: 10px;
+  }
+
+  .km {
+    position: relative;
+    bottom: 7px;
+    left: 6px;
+  }
+
+  .welcome-page__driver-go {
+    margin-top: -2px;
+  }
+
+  .welcome-page__2 {
+    margin-top: 39px;
+    width: auto;
+    justify-content: center;
+    margin-left: 24px;
+  }
+
+  .welcome-page__test-drive {
+    font-size: 16px;
+    line-height: 20px;
+    color: #e51937;
+    margin-left: 15px;
+    margin-top: 3px;
+  }
+
+  .welcome-page__footer-btn {
+    font-size: 16px;
+    line-height: 20px;
+    color: #fff;
+    background: #e51937;
+  }
+
+  .welcome-page__footer {
+    background: #e51937;
+    border: 0;
+    width: 29%;
+    height: 55.21px;
+    margin-right: 0;
+  }
+
+  .welcome-page__button {
+    margin-top: 2%;
+    margin-right: 35px;
+  }
+
+  .arrow {
+    margin-top: 63px;
+  }
+
+}
+
+@media (max-width: 600px) {
+  .welcome-page__footer {
+    margin-right: 3%;
+  }
+
+  .welcome-page__button {
+    margin-left: 3%;
+  }
+
+  .welcome-page__block {
+    margin-left: 3%;
+  }
+
+}
+
+@media (max-width: 520px) {
+  .welcome-page__speed-sec {
+    font-size: 30px;
+  }
+
+  .welcome-page__driver-max {
+    font-size: 30px;
+  }
+
+  .discrip-mob {
+    margin-left: 20px;
+  }
+
+  .welcome-page__block {
+    margin-left: 0;
+  }
+
+  .discrip-sek {
+    bottom: 0;
+  }
+   .km{
+     bottom: 0;
+   }
+
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)